database/migrations/2021_07_19_123858_create_mismatches_table.php

```diff
@@ -18,8 +18,8 @@ public function up()
             $table->id();
             $table->string('statement_guid');
             $table->string('property_id');
-            $table->binary('wikidata_value'); // Saving as BLOB to enable storing JSON seralizations
-            $table->text('external_value');
+            $table->string('wikidata_value', 2048);
```

Comment:
Good catch!! I think it's a good idea to add a comment for future maitainers explaining that the 2048 is an arbitrary number above 1500 which is the string length limit that we decided to set for the project.

Reply:
:pray: Thank you!